headers/functions/arm9.h

```diff
@@ -3,7 +3,15 @@
 
 #include "arm9/itcm.h"
 
-void EntryArm9(void);
+void SvcWaitByLoop(void);
+void SvcSoftReset(void);
+void SvcCpuSet(void);
+void _start(void);
+void MIiUncompressBackward(void);
+void do_autoload(void);
+void StartAutoloadDoneCallback(void);
+void OSiReferSymbol(void);
+void NitroMain(void);
```

Comment:
Do you know if these are (mostly) the only core functions with weird naming? It seems like after these you get into the allocator functions. If these are the only exceptions maybe it might make more sense to implement an exception list for the case checker so we can use the real names.

Reply:
These are not the only ones. I've seen this naming convention in some of the library code, and the number of functions would make an exception list grow very large.

I don't personally like this naming convention of mixing case with <domain>_<action>, but it is a real naming convention used by some C programming circles (e.g., https://github.com/pret/pokeheartgold/issues/213). It may be worth supporting this style outright, rather than making exceptions.